test/box-luatest/gh_8716_pagination_invalid_after_pos_bug_test.lua

```diff
@@ -0,0 +1,29 @@
+local server = require('luatest.server')
+local t = require('luatest')
+
+local g = t.group()
+
+g.before_all(function(cg)
+    cg.server = server:new()
+    cg.server:start()
+    cg.server:exec(function()
+        local s = box.schema.space.create('s')
+        s:create_index('i')
+    end)
+end)
+
+g.after_all(function(cg)
+    cg.server:drop()
+end)
+
+-- Checks that spaces with `_stream` and `_stream_space_cache` can be accessed
+-- from net.box streams.
+g.test_net_box_conn_with_disabled_schema_fetching = function(cg)
```

Comment:
The comment and the test case name are obviously wrong. Also, I'd test pagination over net.box as well.

Anyway, it'd probably be better to add the test cases to the existing files `test/engine-luatest/pagination_test.lua` and `box-luatest/pagination_netbox_test.lua` so as to keep all pagination-related tests together.

Reply:
Fixed.